uint64 serial # Serial ID of control reference
uint64 num_reference # Total number of prediction

float64 dt # Time interval

float64[30] s_ref # Longitudinal distance travelled prediction
float64[30] v_ref # Longitudinal speed prediction
float64[30] a_ref # Longitudinal acceleration prediction